astropy/table/table.py

```diff
@@ -2011,48 +2009,28 @@ def _convert_string_dtype(self, in_kind, out_kind, python3_only):
 
         self._init_from_cols(newcols)
 
-    def convert_bytestring_to_unicode(self, python3_only=False):
+    def convert_bytestring_to_unicode(self):
```

Comment:
Probably needs a changelog entry.

Reply:
I think this kwarg needs to be deprecated instead of just removed so that existing code doesn't just break in 3.0.